ci_jobs_trigger/tests/zstream_trigger/test_zstream_trigger.py

```diff
@@ -66,7 +74,9 @@ def test_process_and_trigger_jobs(mocker, config_dict):
         GET_ACCEPTED_CLUSTER_VERSIONS_PATH,
         return_value=VERSIONS,
     )
-    mocker.patch(TRIGGER_JOBS_PATH, return_value=True)
+    openshift_ci_trigger_job_mocker = mocker.patch(TRIGGER_JOBS_PATH)
+    openshift_ci_trigger_job_mocker.ok = True
```

Comment:
fixture?

Reply:
No, this is not a fixture.
Usually fixture in pytest will be decorated with `@pytest.fixture`